bears/js/JSONFormatBear.py

```diff
@@ -7,17 +7,12 @@
 from coalib.results.Diff import Diff
 from coalib.results.Result import Result
 from coala_utils.param_convertion import negate
+from coalib.misc.Compatability import JSONDecodeError
 
```

Comment:
please can do it in alphabetical order (this is just a preference though ) :sweat_smile: